tests/conftest.py

```diff
@@ -319,6 +345,47 @@ def _create_task(task_name=""):
         yield _create_task
 
 
+@pytest.fixture(scope="function")
+def watch_dog_task(request, init_kwargs, any_x_series_device) -> nidaqmx.system.WatchdogTask:
+    """Gets a task instance."""
+    # set default values used for the initialization of the task.
+    init_args = {
+        "device_name": any_x_series_device.name,
+        "timeout": 0.5,
+    }
+
+    # iterate through markers and update arguments
+    for marker in request.node.iter_markers():
+        if marker.name in init_args:  # only look at markers with valid argument names
+            init_args[marker.name] = marker.args[0]  # assume single parameter in marker
+
+    with nidaqmx.system.WatchdogTask(**init_args, **init_kwargs) as task:
+        yield task
+
+
+@pytest.fixture(scope="function")
+def device(request, generate_device):
+    """Gets a device instance."""
+    device_name = _get_marker_value(request, "device_name", "")
+    return generate_device(device_name=device_name)
+
+
+@pytest.fixture(scope="function")
+def generate_device(init_kwargs):
+    """Gets a factory function which can be used to generate new device objects."""
+
+    def _create_device(device_name=""):
+        return nidaqmx.system.Device(name=device_name, **init_kwargs)
+
+    yield _create_device
+
+
+@pytest.fixture(scope="function")
+def any_x_series_via_grpc(init_kwargs):
```

Comment:
This fixture is redundant with the `grpc_only` marker from https://github.com/ni/nidaqmx-python/pull/336

Reply:
Removed